docs/site/decorators/Decorators_openapi.md

```diff
@@ -430,3 +430,61 @@ export class SomeController {
 When the OpenAPI spec is generated, the `xs-ts-type` is mapped to
 `{$ref: '#/components/schemas/MyModel'}` and a corresponding schema is added to
 `components.schemas.MyModel` of the spec.
+
+## Convenience Decorators
+
+While you can supply a fully valid OpenAPI specification for the class-level
+`@api` decorator, and full operation OpenAPI specification for `@op` and the
+other convenience decorators, there are also a number of utility decorators that
+allow you to supply specific OpenAPI information without requiring you to use
+verbose JSON.
+
+### @tags
+
+[API document](https://loopback.io/doc/en/lb4/apidocs.openapi-v3.tags.html),
+[OpenAPI Operation Specification](https://github.com/OAI/OpenAPI-Specification/blob/master/versions/3.0.0.md#operation-object)
+
+This decorator can currently be applied to class and a class method. It will set
+the `tags` array string property of the Operation Object. When applied to a
+class, it will mark all operation methods of that class with those tags.
+
+Putting `@tags` on a method in a class that already has `@tags` will overwrite
+the tags for that path entry, unless you pass the options `{append: true}`
+
+```ts
+@tags(['Foo', 'Bar'])
```

Comment:
I prefer to have `...names: string[]` so that we can allow `@tags('Foo', 'Bar')`.

Reply:
Looks good!